src/game/setupData.ts

```diff
@@ -0,0 +1,9 @@
+import type { DeckSuit, GameMode, ModelType } from '../utils/constants';
+
+export interface EOPSetupData {
```

Comment:
Why does the interface a different name as the file name? The same is also true for `gameState` etc.

Reply:
No particular reason. Would you prefer to rename the interface to just `SetupData`, or should the file be renamed?

Reply:
As the EOP context is clear in this repository, I would drop the `EOP` prefix.